homestead.root/usr/share/clearwater/infrastructure/scripts/homestead

```diff
@@ -22,6 +22,12 @@ then
     hs_listen_port=$hss_port
 fi
 
+# If homestead_tw_timer is not set, default to the minimum of 6s
+if [[ -z "$homestead_tw_timer" ]]
+then
+  homestead_tw_timer=6
+fi
```

Comment:
A better way to do this is to set the value to 6 unconditionally before the `. /etc/clearwater/config` line. If the config option is specified, it will then overwrite the default value with the configured one.